data/languages.yaml

```diff
@@ -1811,3 +1811,124 @@ ph:
         - kahapon: 1 araw
         - ngayon: 0 segundo
         - isang araw: 2 araw
+
+
+be:
+    name: Belarussian
+
+    skip: ["каля", "у", "ў", "і"]
+
+    monday:
+        - Панядзелак
+        - Пнд
+        - Пн
+    tuesday:
+        - Аўторак
+        - Аўт
+        - Аў
+    wednesday:
+        - Серада
+        - Сер
+        - Ср
+    thursday:
+        - Чацвер
+        - Чцв
+        - Чв
+    friday:
+        - Пятніца
+        - Пят
+        - Пт
+    saturday:
+        - Субота
+        - Суб
+        - Сб
+    sunday:
+        - Нядзеля
+        - Няд
+        - Нд
+
+    january:
+        - Студзеня
```

Comment:
It does, but short names are not widely used.  Anyway, according to linux locale, they are
```
"Стд"
"Лют"
"Сак"
"Крс"
"Тра"
"Чэр"
"Ліп"
"Жнв"
"Врс"
"Кст"
"Ліс"
"Снж"
```